packages/framework-core/src/booster-event-dispatcher.ts

```diff
@@ -77,13 +77,16 @@ export class BoosterEventDispatcher {
       )
       return
     }
-    await Promise.all(
+    const results = await Promise.allSettled(
       eventHandlers.map(async (eventHandler: EventHandlerInterface) => {
         const register = new Register(eventEnvelope.requestID, eventEnvelope.currentUser)
         logger.debug('Calling "handle" method on event handler: ', eventHandler)
         await eventHandler.handle(eventEnvelope.value as EventInterface, register)
         return RegisterHandler.handle(config, logger, register)
       })
     )
+    results
+      .filter((result) => result.status === 'rejected')
+      .forEach((result) => logger.debug(`Event handler execution failed: ${(result as PromiseRejectedResult).reason}`))
```

Comment:
This solution is fine for now, but we should separate event handlers to run in their own lambda each. I created issue #522 to do it in the future